src/megacmd.cpp

```diff
@@ -140,6 +140,7 @@ std::ostringstream & operator<< ( std::ostringstream & ostr, std::wstring const
 #endif
 
 using namespace mega;
+using namespace std;
```

Comment:
This commit was intended so as not to need that there: https://github.com/meganz/MEGAcmd/commit/558392e9541d296aa4dbe851f185ddab1a0cbc93
But we can keep it if required (i wonder why though)